contributor-guide/modules/ROOT/pages/build-systems/b2.adoc

```diff
@@ -1,5 +1,33 @@
 = B2
+:navtitle: B2
 
-*Content under construction.*
+B2 is a high-level build system, developed to use with Boost, designed to make it easy to build pass:[C++] projects. Building projects using B2 often involves no more than a single command line. This section introduces what library developers need to know about preparing a library for B2.
 
-https://www.bfgroup.xyz/b2/[B2]
\ No newline at end of file
+== Configuration Files
+
+The idea is that users build your library with a single command line, and as a developer you specify in configuration files just as much as is necessary to build your library. For example it is not required to tell B2 how to use a certain compiler. B2 supports many compilers out of the box by knowing how to call them and what options they take. If you create a configuration file you just need to tell B2 where to find your source files, what the executable should be called, which compiler B2 should use, and what if any compiler options need set. B2 will then try to find the compiler and automatically build the program.
+
+Writing configurations requires learning a language specifically developed for B2, called _Boost.Jam_. The configurations are stored in files called _Jamfiles_. 
+
+After a compiler is picked to build a program, B2 translates options in the configuration files to the command line options expected by the selected compiler. This makes it possible to write configuration files once and build many times for different platforms and different compilers.
+
+== Supported Languages
+
+B2 can only be used for pass:[C++] and C projects. B2 doesn't support other languages (Java, C#, Python, etc.). It makes more sense to use a different build system (such as xref:build-systems/cmake.adoc[CMake]) for projects with components implemented in other programming languages.
+
+== Example
+
+The following command installs and builds three specific libraries (boost:chrono[], boost:graph[], and boost:thread[]) to the specified path. All dependencies of these three libraries will also automatically be installed and built.
+
+[source,bash]
+----
+b2 install --prefix=C:\Users\YOURNAME\Documents\Boost\Project --with-chrono --with-graph --with-thread
+----
+
+== Summary
+
+B2 is the simplest and lightest weight option to building your Boost library projects.
+
+As a contributor, it is a requirement that B2 can successfully build your library on all supported platforms, using appropriate Jamfiles that you need to write.
```

Comment:
In practice, many contributors don't know B2 very well and copy and paste templates from other projects. I don't know if it's worth mentioning that possibility here.

Reply:
Added links to some examples which could be copy/pasted